README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
The README.md file in your project doesn't reflect your actual project. Please kindly use the following template to write the readme file professionally.

https://github.com/microverseinc/readme-template

Reply:
[ ] nice observation and reveiw :heavy_check_mark: